pybikes/data/ecobici_ba.json

```diff
@@ -0,0 +1,16 @@
+{
+    "instances": [
+        {
+            "tag": "ecobici-ba", 
+            "meta": {
+                "latitude": -34.630665, 
+                "city": "Buenos Aires", 
+                "name": "ecobici-ba", 
```

Comment:
Here we usually list a pretty name (it does not need to be unique). I think it would be `Ecobici`'